src/applications/form-renderer/utils/formConfig.js

```diff
@@ -9,42 +9,69 @@ import {
   digitalFormPhoneAndEmail,
 } from './digitalFormPatterns';
 
-export const selectSchemas = chapter => {
-  switch (chapter.type) {
+const getChapterKey = chapter =>
+  chapter.type === 'digital_form_your_personal_info'
+    ? 'personalInformationChapter'
+    : `chapter${chapter.id}`;
```

Comment:
I had to add a string prefix in order to get the additional chapters to appear _after_ the personal information chapter. One of the downsides of trying to use a Javascript object as an associate array. Even with this hack, we cannot truly guarantee that form pages will always appear in order as long as this fake associative array structure exists.

Reply:
Ideally, the platform team would convert the `chapters` property of form config objects to use [Maps](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) instead of Objects. I don't expect that to occur anytime soon though.

Reply:
We discussed this for clarity and the takeaway is that we need to ensure our object keys are _all_ strings, so that insertion order will be respected. If there are any integer keys, they will be sorted and stored at the "top" of the object. If they are all strings, they will be sorted by insertion order (i.e. new keys will be appended to the end).

It's not awesome that page order in the Veteran-facing form - something that is critically important - is determined by something subject to this nuance. A `Map` would likely be a better data structure.